src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop
+} from "@stencil/core";
+import { UP, DOWN, ENTER, HOME, END, SPACE } from "../../utils/keys";
+import {
+  getElementDir,
+  getElementTheme,
+  getElementProp
+} from "../../utils/dom";
+import { guid } from "../../utils/guid";
+import { chevronLeft16 } from "@esri/calcite-ui-icons";
+import CalciteIcon from "../../utils/CalciteIcon";
+
+@Component({
+  tag: "calcite-accordion-item",
+  styleUrl: "calcite-accordion-item.scss",
+  shadow: true
+})
+export class CalciteAccordionItem {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  @Prop({ reflect: true, mutable: true }) active: boolean = false;
+
+  /** pass a title for the accordion item */
+  @Prop() itemTitle: string;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  //--------------------------------------------------------------------------
+
+  @Event() calciteAccordionItemKeyEvent: EventEmitter;
+  @Event() calciteAccordionItemSelected: EventEmitter;
+  @Event() closeCalciteAccordionItem: EventEmitter;
+  @Event() registerCalciteAccordionItem: EventEmitter;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  //--------------------------------------------------------------------------
+
+  componentDidLoad() {
+    this.itemPosition = this.getItemPosition();
+    this.registerCalciteAccordionItem.emit({
+      item: this.el,
+      position: this.itemPosition
+    });
+  }
+
+  render() {
+    const dir = getElementDir(this.el);
+    const theme = getElementTheme(this.el);
+    const scale = getElementProp(this.el, "scale", "m");
+    const appearance = getElementProp(this.el, "appearance", "default");
+    const iconPosition = getElementProp(this.el, "icon-position", "end");
+    return (
+      <Host
+        theme={theme}
+        dir={dir}
+        scale={scale}
```

Comment:
How come scale/theme/etc aren't documented properties?

Reply:
OH, because its a property of the parent and not the child? 🆗  Maybe it should just get the closest() parent to get that information?

Reply:
It's using the `getElementProp()` util to do that - is there a better way to "look up" the component than that DOM util?

Reply:
Do we want to allow devs to make an accordion of scale `m` with `accordion-item`s of varying scales? I think having it has a prop sort of infers that. 

Wonder if we could just have the props on the accordion itsself and use the inherited scope of CSS variables to style the items?

Reply:
Yes moving to parent now, removing prop duplication on child ✨